workspace_tools/export/codered_lpc4088_cproject.tmpl

```diff
@@ -981,7 +981,7 @@
                             <tool id="com.crt.advproject.gas.exe.release.579950187" name="MCU Assembler" superClass="com.crt.advproject.gas.exe.release">
                                 <option id="com.crt.advproject.gas.arch.1401271875" name="Architecture" superClass="com.crt.advproject.gas.arch" value="com.crt.advproject.gas.target.cm3" valueType="enumerated"/>
                                 <option id="com.crt.advproject.gas.thumb.1024544278" name="Thumb mode" superClass="com.crt.advproject.gas.thumb" value="true" valueType="boolean"/>
-                                <option id="gnu.both.asm.option.flags.crt.637466836" name="Assembler flags" superClass="gnu.both.asm.option.flags.crt" value="-c -x assembler-with-cpp -D__NEWLIB__  -DNDEBUG -D__CODE_RED " valueType="string"/>
+                                <option id="gnu.both.asm.option.flags.crt.637466836" name="Assembler flags" superClass="gnu.both.asm.option.flags.crt" value="-c -x assembler-with-cpp -D__NEWLIB__  -DNDEBUG -D__CODE_RED -DFPU_PRESENT=1 " valueType="string"/>
```

Comment:
In the list you link to there is already a FPU_PRESENT for the Cortex-M4F and not for the Cortex-M4 which is logical. However the project files generated for the LPC4088 for LPCXpresso by the online compiler will have the FPU_PRESENT set for C code and C++ code but not for assembler. To me that indicates that either the project generation routine is wrong or the LPC4088 targets are configured as being both M4 and M4F.